ansible/idr-docker.yml

```diff
@@ -190,6 +190,29 @@
                 USERNAME = "{{ idr_jupyter_omero_user }}"
                 PASSWORD = "{{ idr_jupyter_omero_pass | quote }}"
 
+# - name: download mineotaur data
+#   become: yes
+#   synchronize:
+#     src: "rsync://{{ TBD_omero_host }}/mineotaur/"
+#     dest: "/tmp/mineotaur/"
+
+# - name: change mineotaur to 1000 uid
+#   become: yes
+#   file:
+#     path: "/tmp/mineotaur"
+#     owner: 1000
+
+# - name: enable mineotaur container
+#   docker_container:
+#     name: mineotaur
+#     image: imagedata/mineotaur
+#     entrypoint: java
+#     volumes:
+#       - /tmp/mineotaur:/mineotaur
+#     ports:
+#       - "8080:8080"
+#     command: ["-Xmx8g", "-Djava.security.egd=file:/dev/./urandom", "-jar", "/home/build/src/target/Mineotaur-1.0.1.jar", "-start", "/mineotaur"]
+
```

Comment:
Before you take this further, would you mind moving mineotaur to a separate playbook that's run after `idr-docker.yml` (or maybe included, depending on how things progress)? If it turns out the analysis docker swarm is multipurpose (jupyter and mineotaur) we should start to decouple things. This should also make it easier if we run multiple docker swarms for different purposes.